library_generation/generate_composed_library.py

```diff
@@ -81,7 +81,9 @@ def generate_composed_library(
             library_path=library_path,
         )
         service_version = gapic.proto_path.split("/")[-1]
-        temp_destination_path = f"java-{library.api_shortname}-{service_version}"
+        temp_destination_path = f'java-'
+        temp_destination_path += library.library_name if library.library_name else library.api_shortname
```

Comment:
In `utilities.py`, there's a `get_library_name` method:

https://github.com/googleapis/sdk-platform-java/blob/main/library_generation/utilities.py#L148

Reply:
Thanks, I'll use this one

Reply:
Ah I forgot to push a very similar change